lib/linguist/heuristics.rb

```diff
@@ -127,6 +127,14 @@ def call(data)
         Language["ECL"]
       end
     end
+    
+    disambiguate ".es" do |data|
+      if /^\s*(?:%%|main\s*\(.*?\)\s*->)/.match(data)
+        Language["Erlang"]
+      elsif /(?:\/\/|("|')use strict\1|export\s+default\s|\/\*.*?\*\/)/m.match(data)
```

Comment:
> I've enabled the /m flag so it will span multiple lines.

I missed that :S 

Could you add a test case for these regexp in [`test_heuristics.rb`](https://github.com/github/linguist/blob/master/test/test_heuristics.rb)?
(I'm worried about regressions if we ever change them in the future.)